services/uploads/serverless.yml

```diff
@@ -86,34 +88,48 @@ layers:
 
 functions:
   avScan:
-    handler: src/antivirus.lambdaHandleEvent
+    handler: src/avScan.avScanLambda
     name: ${self:service}-${sls:stage}-avScan
     timeout: 300 # 300 seconds = 5 minutes. Average scan is 25 seconds.
     memorySize: 3008
     layers:
       - !Ref ClamDefsLambdaLayer
-      - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-0-1:1
+      - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-8-0:2
```

Comment:
I'm guessing this is working fine since we're not actually instrumenting anything with OTEL in this service? This is the layer upgrade I want to schedule so that we can bump all our OTEL libs.

Reply:
I’ll back this down, I tried an upgrade when I didn’t know what the problems were yet